### PR TITLE
Kwamena

### PR DESCRIPTION
**Pull request by kwamenaabew**
New pull request.

### CODE REVIEW

File: pages/motherboard.html [hunk 1]
```diff
@@ -1,193 +1,271 @@
...
+            <!-- <div class="grid_container">
+              <div class="grid_span2">
+                <h2 class="heading_nyr_class">Tengdar vörur</h2>
+              </div>
+            </div> -->
+            <ul class=" grid_container">
+              <div class="grid_span2">
+                <h2 class="heading_nyr_class">Tengdar vörur</h2>
+              </div>
+              <div class="column3to11"> </div>
```
Comment (Skatturinn) [inline pages/motherboard.html]: delete this empty div <div class="column3to11"> </div>.

File: pages/robot.html [hunk 2]
```diff
@@ -60,25 +60,127 @@ <h1><strong>Tæknibúðin</strong></h1>
...
+					<!-- <div class="grid_container">
+						<div class="grid_span2">
+							<h2 class="heading_nyr_class">Tengdar vörur</h2>
+						</div>
+					</div> -->
+					<ul class=" grid_container">
+						<div class="grid_span2">
+							<h2 class="heading_nyr_class">Tengdar vörur</h2>
+						</div>
+						<div class="column3to11"> </div>
```
Comment (Skatturinn) [inline pages/robot.html]: empty div

File: styles.css [hunk 3]
```diff
@@ -102,7 +102,10 @@ a {
 }
 
 img {
-	max-width: 100%;
+	grid-column-end: span 1;
```
Comment (Skatturinn) [inline styles.css]: what is this

Comment (Skatturinn) [reply styles.css]: this has to be deleted, is it doing anything?

File: david.css [hunk 4]
```diff
@@ -1,64 +1,386 @@
...
+	grid-template-rows: auto auto minmax(0, 270px);
+	align-items: center;
+	padding: 0 0;
+	justify-items: center;
+	align-items: flex-end;
+	justify-content: center;
+}
+
+.grid_container {
+	opacity: 1;
```
Comment (Skatturinn) [inline david.css]: if it's in styles.css it doesn't need to be in david.css, would be great if you could clean it up

**Review by Add1Padd1 — APPROVED** [review]
All good and well, but we need to resolve the comments Elias made before merge-ing

**Review by Skatturinn — APPROVED** [review]

**Pull request merged**